### PR TITLE
security bump for lodash

### PR DESCRIPTION
**Pull request by daverau-optimizely**
1. Bump for lodash
1. Also includes running `yarn-deduplicate yarn.lock`

Screen Shot 2019-10-03 at 10 49 34 AM 1

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -8407,6 +8407,11 @@ lodash.words@^3.0.0:
   dependencies:
     lodash._root "^3.0.0"
 
+lodash@4.17.13:
```
Comment (quarklemotion) [inline yarn.lock]: Hmmm, this should read `lodash@>=4.17.13` and should not be adding a new version of lodash `4.17.13` to yarn.lock.  I think you may need to revert these yarn.lock changes and re-run `yarn install`

Comment (daverau-optimizely) [reply yarn.lock]: Hey MiFi, thanks for this note. I was kinda confused about how to approach this. When I try a simple `yarn upgrade lodash --latest` it doesn't appear to change lodash:
```oui (drau/yarn-bumpo) → gd
diff --git a/yarn.lock b/yarn.lock
index 10360e38..614a43ab 100644
--- a/yarn.lock
+++ b/yarn.lock
@@ -2445,7 +2445,7 @@ async@^1.4.0, async@^1.5.2:
   resolved "https://registry.yarnpkg.com/async/-/async-1.5.2.tgz#ec6a61ae56480c0c3cb241c95618e20892f9672a"
   integrity sha1-7GphrlZIDAw8skHJVhjiCJL5Zyo=

-async@^2.1.2, async@^2.1.4, async@^2.4.1, async@^2.5.0:
+async@^2.1.2, async@^2.1.4, async@^2.4.1:
   version "2.6.2"
   resolved "https://registry.yarnpkg.com/async/-/async-2.6.2.tgz#18330ea7e6e313887f5d2f2a904bac6fe4dd5381"
   integrity sha512-H1qVYh1MYhEEFLsP97cVKqCGo7KfCyTt6uEWqsTBr9SO84oK9Uwbyd/yCW+6rKJLHksBNUVWZDAjfS+Ccx0Bbg==```

Comment (daverau-optimizely) [reply yarn.lock]: @quarklemotion How would you bump this besides `yarn upgrade lodash --latest`?

**Comment by ayeshakmaz** [comment]
@daverau-optimizely Looks like this was taken care of in https://github.com/optimizely/oui/pull/1227

**Comment by daverau-optimizely** [comment]
@kwalker3690 beat me to it! https://github.com/optimizely/oui/pull/1227

**Pull request closed without merging**